tutorials/animation/animation_tree.rst

```diff
@@ -188,6 +188,47 @@ Transitions also have a few properties. Click any transition and it will be disp
 * *Priority* is used together with the ``travel()`` function from code (more on this later). Lower priority transitions are preferred when travelling through the tree.
 * *Disabled* toggles disabling this transition (when disabled, it will not be used during travel or auto advance).
 
+For better blending
+-------------------
+
+In Godot 4.0+, for blending results consistently, the property values to be blended always have a specific initial value.
```

Comment:
```suggestion
In Godot 4.0+, for blending results consistently, the property values to be blended always have a specific initial value.
```
Grammar feels funky here, but I'm not enough of a natural speaker to provide a suggestion.
Maybe "for blending results to be consistent", or maybe restructure the whole sentence. Not sure. Hope someone else has a suggestion :)

Reply:
@mhilbrunner In Godot 3, when blending was performed during animation playback, the blending result depended on when the blending was performed.

I want to mean that the results of blending always be equal.